### ISSUE TITLE
Optional pickup type not being sent

### ISSUE THREAD

**Issue by mohanenm** [BILDIT-Platform/react-native-bildit-flybuy#31]
The 2.2.3 IOS Flybuy SDK supports sending a pickup type, optionally. Currently, none of the swift methods actually send it through. 

I was able to accomplish this(and still support the others optional params) by implementing this with the example app:

``` @objc(createOrder:withPartnerIdentifier:withCustomerInfo:withPickupWindow:withOrderState:withPickupType:withResolver:withRejecter:)
    func createOrder(siteId: Int,
                     pid: String,
                     customerInfo: Dictionary<String, String>,
                     pickupWindow: Dictionary<String, String>? = nil,
                     orderState: String? = nil,
                     pickupType: String? =  nil,
                     resolve:@escaping RCTPromiseResolveBlock,
                     reject:@escaping RCTPromiseRejectBlock) {
        let info = decodeCustomerInfo(customer: customerInfo)

        func callbackHandler(order: Order?, error: Error?) {
                    if (error == nil && order != nil) {
                        resolve(self.parseOrder(order: order!))
                    } else {
                        reject(error.debugDescription, error?.localizedDescription, error )
                    }
                }
        if (pickupWindow != nil) {
                  let pickupWindowInfo = decodePickupWindow(pickupWindow: pickupWindow)
                  FlyBuy.Core.orders.create(siteID: siteId, partnerIdentifier: pid, customerInfo: info, pickupWindow: pickupWindowInfo, state: orderState ?? "created", pickupType: pickupType ?? "") {
                      (order, error) in callbackHandler(order: order, error: error)
                  }
        } else if (pickupWindow == nil) {
            FlyBuy.Core.orders.create(siteID: siteId, partnerIdentifier: pid, customerInfo: info, state: orderState ?? "created", pickupType: pickupType ?? "") {
                    (order, error) in callbackHandler(order: order, error: error)
                }
        }
    }
```

**Comment by mohanenm** [comment]
in #32

**Comment by mohanenm** [comment]
Fixed in #32
